index.js

```diff
@@ -560,25 +560,20 @@ alexa.app = function(name) {
   // @throws Error when router or expressApp options are not specified
   // @returns this
   this.express = function(options) {
-    if (!options.expressApp) {
-      throw new Error("You must specify an express instance to attach to.");
+    if (!options.expressApp && !options.router) {
+      throw new Error("You must specify an express app or an express router to attach to.");
     }
 
-    if (!options.router) {
-      throw new Error("You must specify an express router to attach.");
-    }
-
-    var defaultOptions = { endpoint: self.name, checkCert: true, debug: false };
+    var defaultOptions = { endpoint: "/" + self.name, checkCert: true, debug: false };
 
     options = defaults(options, defaultOptions);
 
-    var endpoint = "/" + options.endpoint;
-    var router = options.router;
-
-    options.expressApp.use(endpoint, router);
+    // In ExpressJS, user specifies their paths without the '/' prefix
+    var endpoint = options.endpoint;
```

Comment:
@bgjehu This should be `var endpoint = utils.normalizeApiPath(options.endpoint);`